br/pkg/lightning/backend/local/local.go

```diff
@@ -1357,6 +1357,27 @@ func (local *local) ImportEngine(ctx context.Context, engineUUID uuid.UUID, regi
 		return err
 	}
 
+	if len(ranges) > 0 && local.pdCtl.CanPauseSchedulerByKeyRange() {
```

Comment:
What if several engines with range overlapping start importing at the same time, is the new scheduler compatible with this kind of concurrent operations?

Reply:
It's compatible. Different engines will use different label rule id. It guarantees the scheduler is disabled on every engine key range even if overlapping exists.